source/state_representation/include/state_representation/exceptions/InvalidParameterException.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <exception>
+#include <string>
```

Comment:
Why did that work without it? I guess its just cleaner if you put it explicitly?